src/technologies/j.json

```
@@ -638,7 +638,7 @@
       "recurring"
     ],
     "saas": true,
-    "scriptSrc": "*\\.judge\\.me",
+    "scriptSrc": ".*\\.judge\\.me",
```

FYI There is also a case with api2.judge.me on https://anbernic.com/.
A wildcard should work fine.

@tunetheweb could you please merge this.
For me it breaks the web extension, so it may also be the cause of failing detection on the agents.

---

We don’t need the `.*`

```suggestion
    "scriptSrc": "\\.judge\\.me",
```